deploy.sh

```
@@ -0,0 +1,9 @@
+dfx canister stop market-maker-bot-backend --ic
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)